projects/components/select/src/select-data.component.ts

```diff
@@ -114,8 +114,8 @@ export class PsSelectDataComponent<T = any> implements AfterViewInit, OnDestroy
   }
 
   /** Die Fehlernachricht, wenn es beim Items Laden einen Fehler gab */

```

Comment:
these comments should also be in english i guess?

Reply:
done